pinot-controller/src/main/java/com/linkedin/pinot/controller/ControllerStarter.java

```diff
@@ -69,6 +68,7 @@ public ControllerStarter(ControllerConf conf) {
     validationManager = new ValidationManager(validationMetrics, helixResourceManager, config);
     realtimeSegmentsManager = new PinotRealtimeSegmentManager(helixResourceManager);
     segmentStatusChecker = new SegmentStatusChecker(helixResourceManager, config);
+    executorService = Executors.newCachedThreadPool();
```

Comment:
please name the threads so that we can detect in case the number of threads goes large.